profiles/common/modules/features/multisite_forum_community/multisite_forum_community.module

```diff
@@ -7,6 +7,13 @@
 
 include_once 'multisite_forum_community.features.inc';
 
+/**
+ * Implements hook_init().
+ */
+function multisite_forum_community_init() {
+
```

Comment:
Why is there a empty hook_init, it should contain at least a comment with the reason.